src/build/brick.rs

```diff
@@ -318,13 +318,20 @@ impl Baked {
                 })
                 .collect()
         });
-        baked_bricks
+        let search_base = search_alias.with_base();
+        let baked = &baked_bricks
             .iter()
-            .filter(|(baked_alias, _)| alias.matches(baked_alias))
+            .filter(|(baked_alias, _)| search_base.matches(baked_alias))
             .min_by_key(|(brick_alias, _)| brick_alias.0.len())
-            .expect(&format!("no such brick: '{alias}'"))
-            .1
-            .clone()
+            .expect(&format!("no such brick: '{search_base}'")).1;
```

Comment:
```suggestion
            .map(|(_, brick)| *brick)
            .expect(&format!("no such brick: '{search_base}'"));
```
This makes the unpacking of the tuple clearer than postfixing a `.1`